electron-builder.json

```diff
@@ -54,17 +54,9 @@
       {
         "target": "nsis",
         "arch": ["ia32", "x64"]
-      },
-      {
-        "target": "appx",
-        "arch": ["ia32", "x64"]
       }
```

Comment:
@mokagio I was about to suggest adding some comments to mention that the `.appx` is instead generated with a custom config via `make package-win32` so it's properly code-signed, + link to this PR's description for the breadcrumbs…

…but then I just realized this is a `.json` file, which doesn't accept comments 😢 

That being said, [it seems that `electron-builder` supports configuration files in YAML (or js) format](https://www.electron.build/configuration/configuration.html). So WDYT of migrating the project to a `electron-builder.yml` file instead of `.json` in a future PR, so we can start adding such comments to it?

Reply:
Sure, 100% agree. I thought I had written it somewhere as a TODO myself, but I couldn't find a reference to share here.

Every time I see a JSON used for config these days my first instinct is to ask if we can make it into a YAML. Being able to comment and use anchors and aliases makes the config work simpler.